lib/io.js

```diff
@@ -123,7 +123,17 @@ module.exports = app => {
   }
 
   if (config.redis) {
-    app.io.adapter(redis(config.redis));
+    // Use ioredis to support sentinel mode if redis config
+    // has 'sentinels' item.
+    if (config.redis.sentinels) {
+      const Redis = require('ioredis');
```

Comment:
Instead of creating the Redis instance directly, I prefer `app.redis.createInstance({})`. So, we can remove the `require('ioredis');`

Reply:
app.redis.createInstance will execute createClient method which has code execute `client.info` as following,but subcribe status doesn't support `info` command
```js
//egg-redis/lib/redis
//in function createClient
app.beforeStart(async () => {
    const result = await client.info();
    const index = count++;
    app.coreLogger.info(`[egg-redis] instance[${index}] status OK, redis currentTime: ${result[0]}`);
  });
```

Reply:
I noticed that you have raised the sentinel support PR at https://github.com/eggjs/egg-redis/pull/19/files
We should properly wait for that.